Assets/Scripts/Player/PlayerActions/SlideAction.cs

```diff
@@ -67,6 +68,8 @@ public class SlideAction : PlayerAction{
         pathCreator = pc;
         initialEnemyBonus = enemyBonus;
 
+        slideableObjectScaleOffset = 1f / other.transform.localScale.y;
```

Comment:
only factoring the y component of the scale is an incomplete solution because the length of the path can be increased when increasing the x scale too

Reply:
This and the above comment about increasing the slide speed from 15 ->100, makes me think that using the scale to scale down the speed is not the right approach. Another suggestion is moving the player position in the direction of the path then getting the closest distance at the path.

Reply:
Fixed!